broker-daemon/interchain-router/get-preimage.js

```diff
@@ -83,6 +83,8 @@ async function getPreimage ({ params, send, onCancel, onError, ordersByHash, eng
   const { inboundSymbol, inboundFillAmount, outboundSymbol, outboundFillAmount, takerAddress } = order
   const [ expectedSymbol, actualSymbol, expectedAmount, actualAmount ] = [ inboundSymbol, symbol, inboundFillAmount, amount ]
 
+  logger.debug(`Found routing entry for swap ${swapHash}`)
```

Comment:
should we log `order` or is that too much information for this log?

Reply:
I think that's too much information